.circleci/config.yml

```diff
@@ -92,7 +92,7 @@ jobs:
           command: yarn install --immutable
       - run:
           name: Bootstrap
-          command: yarn bootstrap --core
+          command: yarn bootstrap --core --optimize
```

Comment:
So by default when executing the bootstrap command we do NOT optimise.

This but in the CI we do do it.